src/msg/async/rdma/RDMAStack.h

```diff
@@ -235,6 +237,20 @@ class RDMAConnectedSocketImpl : public ConnectedSocketImpl {
       active = false;
     }
   };
+  class C_handle_connection_init : public EventCallback {
```

Comment:
no need to define this callback in the header file. please define it in an anonymous namespace in `RDMAConnectedSocketImpl.cc`.

Reply:
don't need keep C_handle_connection_init in the  same format as C_handle_connection?

Reply:
good idea! better off moving `C_handle_connection` to the .cc file as well, please do this in a separated commit.